### PR TITLE
Added Ashtapada

### PR DESCRIPTION
**Pull request by Akshat12210**
- Added a game in strategy section named "Ashtapada"

Submitting this PR means you have:

- [x] searched the [readme][readme-url] file and searched the [open pull requests][pr-url] (no duplicates please)
- [x] and will follow [the desired format][format-url]
- [x] used image links that resolve correctly, are appropriately sized, and are served over HTTPS

Once your PR is created, it will trigger a build a GitHub Action (Awesome-Lint). This includes a format linting process, so if there are any issues with the build, please address them.

Do you have further questions? Check the [contribution guidelines][contributing-url] or [open an issue][new-issue-url].

When you're ready, check off the above acknowledgements and continue with your Pull Request... and thanks!

[pr-url]: https://github.com/edm00se/awesome-board-games/pulls
[readme-url]: https://github.com/edm00se/awesome-board-games/edit/main/readme.md
[format-url]: https://github.com/edm00se/awesome-board-games/blob/main/formatting.md
[contributing-url]: https://github.com/edm00se/awesome-board-games/blob/main/contributing.md
[new-issue-url]: https://github.com/edm00se/awesome-board-games/issues/new

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -598,6 +598,17 @@
...
 
+### [Ashtapada](https://en.wikipedia.org/wiki/Ashtapada)
+
+> Ashtapada, an ancient game originating in India, is played on an 8x8 grid featuring marked spaces. The game can be played by two to four players, with the objective being to enter and exit both pieces strategically. Players use cowrie shells to determine movement. If a player lands on an opponent's piece, it's sent off the board for re-entry, except on 'crosscut' spaces, where pieces remain safe.
+
+![Ashtapada game image](https://cf.geekdo-images.com/kg1eP5Vdj6LoFqQ2E2Kxew__opengraph/img/lykEixPPGFGql26yrwEYvhj3XMo=/fit-in/1200x630/filters:strip_icc()/pic55539.jpg)
+
+| Players | Min. Age | Time   |
+| ------: | -------: | -----: |
+|   2–4 |          8 | 30–60 Min |
```
Comment (edm00se) [inline readme.md:609]: Where are you getting the stats from? BGG shows 2 players only, neither they nor wikipedia have any age or time frame ranges.

Comment (Akshat12210) [reply readme.md:609]: Hello,
One of the resources I followed is [Ashtapada](https://a4gamescompany.wordpress.com/ashtapada/). Here, it was mentioned regarding the number of players. When talking about age and time, it's just an estimate I have gathered from a few other resources.
What should I write if I don't have a proper estimate for age and time?

Thanks for your time and consideration.

Comment (edm00se) [reply readme.md:633]: Thank you. I'm considering how best to move forward. It sounds like it's an ancient enough game to not have a lot in the way of documentation 😆

File: readme.md [hunk 2]
```diff
@@ -637,6 +637,17 @@
...
+
```
Comment (edm00se) [inline readme.md:649]:
```suggestion
```

**Review by edm00se — APPROVED** [review]
Thanks for contributing!

refs #269

**Pull request merged**